docker.d/configs/production/worker.json

```diff
@@ -0,0 +1,81 @@
+{
+    "work_dir": "/app/work",
+    "artifact_dir": "/app/artifacts",
+    "aiohttp_max_connections": 10,
+    "checksums_digests": ["sha512", "sha256"],
+    "blobs_needing_prettynaming_contents": [
+        "target.test_packages.json"
+    ],
+    "taskcluster_scope_prefix": "project:releng:beetmover:",
```

Comment:
Hm, not all the projects start with this prefix. Some do with `project:comm: or `project:mobile` (see [here](https://github.com/mozilla-releng/build-puppet/blob/eec2193b8ff53db49f64a23a5f87cbe13f60d48c/modules/beetmover_scriptworker/manifests/settings.pp)). Does it matter?

Reply:
We are going to use different configs for those.

Reply:
We'll just use different configs for different prod environments.